.github/workflows/tests.yml

```
@@ -0,0 +1,64 @@
+        with:
+          node-version: '22.x'
+
+      - name: Enable corepack
+        run: corepack enable
+
+      - name: Install dependencies
+        if: steps.filter.outputs.relevant == 'true'
+        uses: bahmutov/npm-install@v1
+        with:
+          install-command: yarn install --immutable
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)